File: output/deepseek_r1/avatar/direct_translation/java/python/successful_translation/codeforces_469_A.py
n = int(input())
p_part = list(map(int, input().split()))
p = p_part[0]
x = p_part[1:p+1]
q_part = list(map(int, input().split()))
q = q_part[0]
y = q_part[1:q+1]
combined = set(x + y)
print("I become the guy." if all(i in combined for i in range(1, n+1)) else "Oh, my keyboard!")

File: output/deepseek_r1/avatar/direct_translation/python/java/successful_translation/codeforces_469_A.java
import java.util.*;

public class codeforces_469_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int maxLevel = Integer.parseInt(scanner.nextLine());
        String[] xParts = scanner.nextLine().split(" ");
        String[] yParts = scanner.nextLine().split(" ");
        
        Set<Integer> xSet = new HashSet<>();
        for (int i = 1; i < xParts.length; i++) {
            xSet.add(Integer.parseInt(xParts[i]));
        }
        
        Set<Integer> ySet = new HashSet<>();
        for (int i = 1; i < yParts.length; i++) {
            ySet.add(Integer.parseInt(yParts[i]));
        }
        
        if (xSet.contains(0)) {
            xSet.remove(0);
        } else if (ySet.contains(0)) {
            ySet.remove(0);
        }
        
        Set<Integer> union = new HashSet<>(xSet);
        union.addAll(ySet);
        
        System.out.println(union.size() == maxLevel ? "I become the guy." : "Oh, my keyboard!");
    }
}
